docs/tutorials/03_ground_state_solvers.ipynb

```diff
@@ -241,15 +257,15 @@
      "text": [
       "=== GROUND STATE ENERGY ===\n",
       " \n",
-      "* Vibrational ground state energy (cm^-1): (1.1e-11-0j)\n",
+      "* Vibrational ground state energy (cm^-1): (1e-11+0j)\n",
       "The number of occupied modals is\n",
       "- Mode 0: [0.0, 0.0, 0.0, 0.0]\n",
       "\n",
       "\n",
       "\n",
       "=== GROUND STATE ENERGY ===\n",
       " \n",
-      "* Vibrational ground state energy (cm^-1): (2553.464851175474+0j)\n",
+      "* Vibrational ground state energy (cm^-1): (2555.376153240112+0j)\n",
```

Comment:
I am curious why this value changed so significantly.. I don't think it's this PRs fault but maybe we should look into that!

Reply:
I ran locally from MacOS but those are similar to results we have been getting from 'main'. Looking at the tutorials run from a recent nightly CI (Ubuntu):
```
Vibrational ground state energy (cm^-1): (2555.376153240098-0j)
```

Reply:
Yeh this could have happened during the migration from Aqua. But I think nonetheless we should investigate when/why this differences started to occur. Maybe a bisection will tell :thinking: (But this is unrelated to this PR for sure!)